examples/gno.land/r/demo/markdown_test/markdown.gno

```diff
@@ -175,6 +173,403 @@ If markdown is too limiting, you can just insert your own <strike>crazy</strike>
+
+Content 1
+
+:::box/
+:::box
+
+### Col 2
+
+Content 2
+
+:::box/
+:::columns/
+
+:::columns (3)
+:::box
+
+### Col 1
+
+Content 1
+
+:::box/
+:::box
+
+### Col 2
+
+Content 2
+
+:::box/
+:::box
+
+### Col 3
+
+Content 3
+
+:::box/
+:::columns/
+
+:::columns (4)
+:::box
+
+### Col 1
+
+Content 1
+
+:::box/
+:::box
+
+### Col 2
+
+Content 2
+
+:::box/
+:::box
+
+### Col 3
+
+Content 3
+
+:::box/
+:::box
+
+### Col 4
+
+Content 4
+
+:::box/
+:::columns/
+
+:::columns (5)
+:::box
+
+### Col 1
+
+Content 1
+
+:::box/
+:::box
+
+### Col 2
+
+Content 2
+
+:::box/
+:::box
+
+### Col 3
+
+Content 3
+
+:::box/
+:::box
+
+### Col 4
+
+Content 4
+
+:::box/
+:::box
+
+### Col 5
+
+Content 5
+
+:::box/
+:::columns/
+
+:::columns (6)
+:::box
+
+### Col 1
+
+Content 1
+
+:::box/
+:::box
+
+### Col 2
+
+Content 2
+
+:::box/
+:::box
+
+### Col 3
+
+Content 3
+
+:::box/
+:::box
+
+### Col 4
+
+Content 4
+
+:::box/
+:::box
+
+### Col 5
+
+Content 5
+
+:::box/
+:::box
+
+### Col 6
+
+Content 6
+
+:::box/
+:::columns/
+
+Button component
+----------------
+
+Button component is used as CTA (for a link) or to display a button element that you will be able to use to trigger an action or event through Javascript.
+
+:::button (https://gno.land)
+Link button
+:::button/
+
+:::button
+State button
+:::button/
+
+Accordion component
+-------------------
+
+Accordions display a list of high-level options that can expand/collapse to reveal more information. It takes the button text as argument.
+
+*Todo: add argument to set the state at first (open or closed)*
+
+:::accordion (Accordion open/close button)
+Accordion content
+:::accordion/
+
+
+Tabs component
+--------------
+
+A component to create accessible tabs. It takes the button text as arguments.
+
+:::tabs (1st tab button text)(2nd tab button text)
+:::box
+
+## 1st Tab Title
+
+1st tab Content
+:::box/
+
+:::box
+
+## 2nd Tab Title
+
+2nd tab Content
+:::box/
+:::tabs/
+
+
+Alert component
+---------------
+
+Alerts are used to communicate a state that affects a system, feature or page. The component take the alert style as argument: "warning", "info", "danger", "success".
+
+:::alert (warning)
+Warning content
+:::alert/
+
+:::alert (info)
+Info content
+:::alert/
+
+:::alert (danger)
+Danger content
+:::alert/
+
+:::alert (success)
+Success content
+:::alert/
+
+Breadcrumb component
+--------------------
+
+Breadcrumbs is a navigation pattern that helps users understand the hierarchy of a website. It requires to be filled by an ordered list.
+
+:::breadcrumb
+
+1. [Home](https://gno.land)
+2. [Foo](https://gno.land)
+3. [Bar](https://gno.land)
+
+:::breadcrumb/
+
+Dropdown component
+------------------
+
+Dropdowns are toggleable, contextual overlays for displaying lists of links and more. It requires to be filled by a list and takes the dropdown text button as argument.
+
+:::dropdown (Dropdown - click here)
+
+1. [Home](https://gno.land)
+2. [Foo](https://gno.land)
+3. [Bar](https://gno.land)
+
+:::dropdown/
```

Comment:
Same question here. why ordered list?

The numbering doesn't affect the order here too

Reply:
I've switch for unordered list here. Both are valid anyway.